.github/workflows/CI.yml

```diff
@@ -97,4 +97,30 @@ jobs:
       - name: Testing CLI (Runs both unit and integration tests)
         run: |
           pip install .
```

Comment:
Can we move installing `test-requirements` to under `Testing` step here and remove installing it  from `Install dependencies` step instead of adding new task for it?

Reply:
We can but I kept it that way to separate the output. Changed it though